app/Models/Category.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Category extends Model
+{
+    protected $table = 'category';
+    protected $primary_key = 'id';
+    protected $guarded = [];
+
+    public function post(){
+        return $this->hasMany("App\Models\Post", "id");
+    }
```

Comment:
```suggestion
    public function post()
    {
        return $this->hasMany(Post::class, "id");
    }
``` use App\Models\Post ở trên cùng nhé;